ckanext/xloader/plugin.py

```diff
@@ -211,4 +211,6 @@ def get_helpers(self):
         return {
             "xloader_status": xloader_helpers.xloader_status,
             "xloader_status_description": xloader_helpers.xloader_status_description,
+            "is_xloader_format": xloader_helpers.is_xloader_format,
```

Comment:
I feel like these two helpers could be combined, since they're really part of the same "does this resource qualify?" check.

Reply:
Helpers and conditions combined into one helper